conf/provisioning/dashboards/nuodb/memory.json

```diff
@@ -1174,24 +1151,27 @@
       "links": [],
       "nullPointMode": "null",
       "options": {
-        "dataLinks": []
+        "alertThreshold": true
       },
       "percentage": false,
+      "pluginVersion": "7.3.2",
       "pointradius": 5,
       "points": false,
       "renderer": "flot",
       "seriesOverrides": [
         {
+          "$$hashKey": "object:180",
           "alias": "/reload.*/i",
-          "stack": "A"
+          "stack": "A",
+          "transform": "negative-Y"
         }
       ],
       "spaceLength": 10,
       "stack": true,
       "steppedLine": false,
       "targets": [
         {
-          "alias": "loaded[$tag_pid]  $tag_host",
+          "alias": "loaded(host: $tag_host, db: $tag_db, nodeid: $tag_nodeid, startid: $tag_startid)",
```

Comment:
Can we have the same alias as in the other panels?

Reply:
> I haven't checked them all but annotations are behaving differently on this dashboard and I think the other two "top views" - toggle and filter don't appear to work because "Annotations & Alerts" is enabled by default. Was that intentional?